roles/prometheus-node-exporter/tasks/main.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
The role name should not use `-` (dash). [see](https://galaxy.ansible.com/docs/contributing/creating_role.html#role-names)

> Role names are limited to lowercase word characters (i.e., a-z, 0-9) and ‘_’. No special characters are allowed, including ‘.’, ‘-‘, and space. During import, any ‘.’ and ‘-‘ characters contained in the repository name or role_name will be replaced with ‘_’.

Reply:
Removed prometheus-node-exporter role